src/ui/design-system/src/lib/Pages/HomePage/index.tsx

```diff
@@ -24,42 +22,29 @@ export type HomePageProps = SocialLinksSignupProps & {
   concepts?: TutorialCardProps[];
   homepageStartProjectData: HomepageStartItemProps[];
   contentNavigationListItems: ContentNavigationListProps;
-  editPageUrl?: string;
   flips: FlipsProps;
-  tools: ToolCardProps[];
   upcomingEvents: UpcomingEventsProps;
 };
 
 const HomePage = ({
-  concepts,
   homepageStartProjectData,
   contentNavigationListItems,
   discordUrl,
-  editPageUrl,
   flips,
   githubUrl,
-  tools,
   upcomingEvents,
 }: HomePageProps): JSX.Element => {
   return (
     <PageBackground gradient="home">
       <LandingHeaderHome
-        description="Discover the developer ecosystem and master the Flow blockchain"
+        description="Start your Flow journey now with the Flow Playground, or explore our comprehensive guides, resources, and references to kickstart your Flow development."
```

Comment:
@nialexsan  nit: Can you please add a link "Flow Playground" 
I would suggest internally if we have an anchor to it's description or page and externally to the playground itself if necessary.

Reply:
we don't have a dedicated page for the playground, probably we should
I've put a direct link to https://play.flow.com/